src/Helper/DateTimeHelper.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\IntlBundle\Helper;
+
+use Sonata\IntlBundle\Locale\LocaleDetectorInterface;
+use Sonata\IntlBundle\Timezone\TimezoneDetectorInterface;
+
+/**
+ * DateHelper displays culture information. More information here
+ * http://userguide.icu-project.org/formatparse/datetime.
+ *
+ * NEXT_MAJOR: Remove all \DateTime hints from PHPDoc
```

Comment:
Since a datetime is already a DatetimeInterface, why cant it be already remove from the phpdoc ?